messenger.go

```diff
@@ -125,8 +126,7 @@ func (r responseSender) Send(binaryReply []byte) {
 	// TODO: detect multiple responses on the same message and spam the log
 	// about it.
 
-	// It would be preferable to replace this with channels so sending
-	// doesn't have to wait on the main loop to come around.
+	glfw.PostEmptyEvent()
```

Comment:
Before:
```
flutter: 00:00 +0: Test Battery Plugin result
Send took 25.112162ms
flutter:           -- Battery level at 55 % .
flutter: 00:00 +1: Complex plugin Test StandardMethodCodec array of map
Send took 25.079956ms
flutter: 00:00 +2: Complex plugin Test golang InvokeMethodWithReply
Send took 25.104811ms
flutter: 00:03 +3: Complex plugin Custom errors
go-flutter: handler for method 'getError' on channel 'instance.id/go/data' returned an error: Some error
Send took 25.076353ms
flutter: 00:03 +4: Complex plugin Test golang CatchAllHandleFunc
Send took 25.06359ms
flutter: 00:03 +5: (tearDownAll)
flutter: 00:03 +6: All tests passed!
Send took 25.213678ms
go-flutter: closing application
flutter: 00:03 +6: All tests passed!

```


After:
```
flutter: 00:00 +0: Test Battery Plugin result
Send took 53.526µs
flutter:           -- Battery level at 55 % .
flutter: 00:00 +1: Complex plugin Test StandardMethodCodec array of map
Send took 77.784µs
flutter: 00:00 +2: Complex plugin Test golang InvokeMethodWithReply
Send took 72.655µs
flutter: 00:03 +3: Complex plugin Custom errors
go-flutter: handler for method 'getError' on channel 'instance.id/go/data' returned an error: Some error
Send took 53.314µs
flutter: 00:03 +4: Complex plugin Test golang CatchAllHandleFunc
Send took 209.681µs
flutter: 00:03 +5: (tearDownAll)
Send took 82.624µs
go-flutter: closing application
flutter: 00:03 +6: All tests passed!

```

Reply:
Wow that's a serious speed improvement! Nice :rocket: